package.json

```diff
@@ -54,6 +54,7 @@
   "scripts": {
     "prepublish": "gulp prepublish",
     "pretest": "gulp pretest",
+    "prestart": "webdriver-manager start > /dev/null 2>&1 &",
```

Comment:
Sure (you should be able to close it yourself via the UI if I remember correctly, but i'm happy to do it as well 😄 ). Thanks!